src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -9239,10 +9239,10 @@ namespace WaterThermalTanks {
 			if ( WaterThermalTank( WaterThermalTankNum ).FirstRecoveryDone && WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel > 0.0 ) {
 				// Calculate Recovery Efficiency based on energy used to recover from the first draw
 				// FirstRecoveryFuel is recorded inside the CalcWaterThermalTank subroutine
-				RecoveryEfficiency = DrawMass * CPHW( 57.2222 ) * ( 57.2222 - 14.4444 ) / WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel;
+				RecoveryEfficiency = DrawMass * CPHW() * ( 57.2222 - 14.4444 ) / WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel;
 
 				// Calculate Energy Factor based on total energy (including parasitics) used over entire test
-				EnergyFactor = TotalDrawMass * CPHW( 57.2222 ) * ( 57.2222 - 14.4444 ) / FuelEnergy;
+				EnergyFactor = TotalDrawMass * CPHW() * ( 57.2222 - 14.4444 ) / FuelEnergy;
```

Comment:
@Myoldmopar Since the temps are hard values here, this must be some standard rating calc?  So, a comment would be in order here.  As for all the other uses of CPHW and CPCW, you could put comments at each location, or a blanket comment in the declaration of the CPW constant value that says for water coils use entering temperature and steam coils use outlet temp (?)  That's what I see based on a quick scan here.